example-playbooks/async_extra/library/async_status_id.py

```diff
@@ -211,6 +198,22 @@ def main():
                                          " probably async_wrapper.py wrapper unexpectedly died." +
                                          " check how " + os.path.join(request.async_dir, request.jid) + " is doing." +
                                          " if it is failed, delete the soft link (alias) to it.", **response.data())
+    else:
+        if request.mode == 'cleanup':
+            if request.alias:
+                os.unlink(request.alias_path)
+            elif request.jid:
+                os.unlink(request.log_path)
+            module.exit_json(erased=request.log_path, **response.data())
+
+        if request.mode == 'full_cleanup':
+            if request.alias:
+                os.unlink(request.alias_path)
+            os.unlink(request.log_path)
+            module.exit_json(erased=request.log_path, **response.data())
+
+        if response.killed:
+            module.exit_json(**response.data())
```

Comment:
These 2 lines can be removed now, since `exit_json` is also being called on line 218

Reply:
simplified (in addition with a few more lines).